src/PropTypeShapes.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from "prop-types";
+
+export const ImagePropType = PropTypes.shape({
```

Comment:
Something I've been thinking about is whether we actually want to define these with `PropTypes.shape()` wrapped around it. When we do this, we lose the ability to reference the individual fields outside of the `shape()`, since they come "prewrapped" with `shape()`.

For a concrete example, take `ParagraphBlock` in `ThreeParagraphBlock`. Its prop types are the usual `title` and `description` prop type that we have, but because its props are directly listed there, we can't make use of the `TitleDescriptionPropType` because it comes with a `shape()`.

```js
// What we have today
ParagraphBlock.propTypes = {
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
};

// Invalid, since this is actually short for PropTypes.shape({ title: ..., description: ...})
ParagraphBlock.propTypes = TitleDescriptionPropType;
```

If we instead defined `TitleDescriptionPropType` as just the raw object without the `PropTypes.shape()`, then we could reuse it here while still retaining the ability to wrap it with a `shape()` in the places that we need it. The downside to this approach is that the other places become more verbose.

Another benefit to this approach of using plain objects is that we could actually mix multiple objects together. I wouldn't recommend doing this very often, since it could get hard to read, but in theory, we could write things like this:

```js
const MyImageWithText = () => {}

MyImageWithText.propTypes = {
  ...ImagePropType,
  ...TitleDescriptionPropType,
};
```

What do you think?